src/project/project.schedule.ts

```diff
@@ -20,4 +22,17 @@ export class ScheduleService extends NestSchedule {
       throw new Error(e);
     }
   }
+
+  @Cron('*/20 * * * *', {
```

Comment:
Comments to explain this? Is this "every 20 minutes"? Where does startTime come from?

Reply:
All good questions. However, some of it can be inferred.

the `startTime` option is part of the Cron decorator, which is imported from `nest-schedule`. I like comments but I think we need to be comfortable with looking up documentation when we can at least trace things back to a package.

I will add a note about what that cron schedule expression translates though although it should be easy enough to look it up.

Reply:
Thanks! I thought a comment here for clarity on scheduled time would be preferred as a team because the decorator API in itself is not self-explanatory. (Contrast this to a signature maybe like `@Cron({ interval: 20, units: 'minutes'  }`  I was able to look it up but it took a couple minutes. Thought it might be nice for someone who rarely touches this part of the code, or a PM who just needs to quickly find out our sync frequency.

Also in this case the decorator options threw me off since Nest's Task Scheduling docs has no mention of them, nor `nest-schedule` (https://docs.nestjs.com/techniques/task-scheduling#task-scheduling)

Reply:
@godfreyyeung thank you! That degree of investigation into these PRs would really help me understand better in the future